docs/fundamentals/client/client.md

```diff
@@ -668,7 +663,26 @@ with Flow().add() as f, open('output.txt', 'w') as fp:
     )
 ```
 
+````{admonition} What errors can be handled by the callback
+:class: caution
+Callback can handle errors that are caused by Executors raising an Exception.
+It will not receive exceptions or from the Gateway having connectivity errors with the Executors or between the Client and the Gateway.
+````
 
+### Continue streaming when an error occurs
+
+`client.post()` accepts `continue_on_error` parameter. When set to True, `Client` will keep trying to send the following requests. `continue_on_error` parameter will only apply
+to Exceptions caused by Executor exceptions, but in case of network connectivity issues, an Exception will be raised.
+
+### Transient fault handling with retries
+
+`client.post()` accepts `max_attempts`, `initial_backoff`, `max_backoff` and `backoff_multiplier` parameters to control the capacity to retry requests, using exponential backoff strategy, when a transient connectivity error occurs.
+This can help to overcome transient network connectivity issues. 
+
+The `max_attempts` determines the number of sending attempts, including the original request.
+The `initial_backoff`, `max_backoff`, and `backoff_multiplier` parameters determine the randomized delay in seconds before retry attempts.
```

Comment:
IMO this is still unclear in the documentation what this parameters actually do.

Maybe we should include a small example and explain when the client will retry to send the requests with respect to the given parameters

Reply:
I added the formula